Dockerfile

```diff
@@ -1,18 +1,54 @@
-FROM silintl/php7:7.2
-MAINTAINER Phillip Shipley <phillip_shipley@sil.org>
-
-ENV REFRESHED_AT 2020-04-07
+FROM silintl/php7-apache:7.4.19
+LABEL maintainer="Phillip Shipley <phillip_shipley@sil.org>"
 
 RUN apt-get update -y && \
-    apt-get install -y php-memcache && \
-    apt-get clean
+    apt-get install -y \
+        git \
```

Comment:
Why is `git` installed?

Reply:
Good question. I ran into this error from composer: "git was not found in your PATH, skipping source download" and that was the only way I could find to get past it. Have you seen that before?